sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala

```diff
@@ -83,7 +83,10 @@ package object dsl {
     def >= (other: Expression): Predicate = GreaterThanOrEqual(expr, other)
     def === (other: Expression): Predicate = EqualTo(expr, other)
     def <=> (other: Expression): Predicate = EqualNullSafe(expr, other)
-    def !== (other: Expression): Predicate = Not(EqualTo(expr, other))
+    def =!= (other: Expression): Predicate = Not(EqualTo(expr, other))
+
+    @deprecated("!== does not have the same precedence as ===, use =!= instead", "1.7.0")
+    def !== (other: Expression): Predicate = this =!= other
```

Comment:
Oh, I missed that. Are you suggesting I should change without deprecation or leave the oerator name as-is?